### PR TITLE
Additional content

### PR DESCRIPTION
**Pull request by joshuamil**
Added new content for Regular Expressions, Shortcuts and then added navigation from the main page.

### CODE REVIEW

File: REGEX.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Regular Expression (RegEx) Resources
+
+> "Some people, when confronted with a problem, think 
+“I know, I'll use regular expressions.”   Now they have two problems." - Jamie Zawinski
+
+Regular Expressions are a powerful meta-syntax for describing search patterns. They can be difficult to understand when you first get started, but once you get the basics, you will find myriad uses for them in your day-to-day work. Everything from your command line (grep, sed, awk) to your editor (Atom, VS Code, Sublime, etc.) to your language-of-choice (Ruby, JavaScript, PHP, Python, Java, Perl, etc.) supports RegEx. You'll find that your programs and your workflow are enhanced greatly by an understanding of this powerful search syntax. 
+
+
+## Regular Expression Tools
+* **[RegEx101.com](https://regex101.com/)** - Handy tool for testing your RegEx in real-time
```
Comment (binarymason) [inline REGEX.md]: regex101 FTW!

Maybe say testing _and explaining_?  I think that is one area that makes regex101 shine.

File: SHORTCUTS.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* `/string` - Search forward for string
+* `?string` - Search backward for string
+* `n` - Search for next instance of string
+* `N` - Search for previous instance of string
+
+## Custom Shortcuts in macOS / Unix / Linux
+You can create `alias` commands in your `~/.bash_profile` file in macOS to create your own shortcut commands. For example, look at the following shortcut ideas:
+
+```
+alias ll='ls -lG'
```
Comment (binarymason) [inline SHORTCUTS.md:41]: 👀 @kaityhallman 👍

**Review by binarymason — APPROVED** [review]
@joshuamil this is great.  Thank you!

File: REGEX.md (hunk 1 above)
Comment (joshuamil) [reply REGEX.md]: Great call-out, I made the change!

**Pull request merged**